### PR TITLE
Pull request

### PR DESCRIPTION
**Pull request by robertschickler**

### CODE REVIEW

File: users/schicklr/index.md [hunk 1]
```diff
@@ -5,8 +5,39 @@ author: Robert Schickler
 ---
 Robert Schickler
 ================================
-I like pizza.
+<img src="images/schickler.jpg">
```
Comment (harterj) [inline users/schicklr/index.md:8]: ``
where X and Y are appropriate picture dimensions

File: users/schicklr/index.md [hunk 2]
```diff
@@ -5,8 +5,39 @@ author: Robert Schickler
...
+***
+
+### Personal interests
+I am a huge fan of pop culture and would be a valuable asset to any trivia team.  I love TV/film/music and partake when time permits.
+
+***
+
+### Curriculum Vitae
+
+<a href="files/schicklerCV.pdf">Click here for CV</a>
```
Comment (harterj) [inline users/schicklr/index.md:41]: `[CV goes here](./files/schicklerCV.pdf)`

this is the styling for markdown, you have HTML in there. You can change "CV goes here" to whatever you want, it'll be a clicky link

File: users/schicklr/index.md (hunk 1 above)
Comment (harterj) [reply users/schicklr/index.md:8]: If you want the exact dimensions your picture has, you can omit `height="X" width="Y"`. Just consider how big it'll be on the page. For instance, my picture dimensions are 500x500

**Comment by albeanth** [comment]
@robertschickler  if you want to invoke the changes Jackson suggests, feel free. Either way, I approve for it to be merged @rtrp/admins .

File: users/schicklr/index.md (hunk 1 above)
Comment (robertschickler) [reply users/schicklr/index.md:8]: I wanted to use the default picture size.

File: users/schicklr/index.md (hunk 2 above)
Comment (albeanth) [reply users/schicklr/index.md:41]: considering this is HTML vs Markdown syntax, I'm going to merge this in.

**Comment by dougwoods3** [comment]
I like Jackson's edits, then :shipit:

File: users/schicklr/index.md (hunk 2 above)
Comment (robertschickler) [reply users/schicklr/index.md:41]: I was emulating links you had on your index file.  The CV seems to load fine for me in HTML.

**Comment by dougwoods3** [comment]
@robertschickler It looks pretty sharp.

**Comment by robertschickler** [comment]
Thx bebe

Now how does the link on the main sitepage get updated?  It still shows me liking pizza and pie, which is fine but not as professional as I'd like to appear to the public.

**Comment by dougwoods3** [comment]
@rtrp/admins @robertschickler What do you guys think about having personal information on our CV/resume for all the world to see? Robert, I noticed your address and cell phone is why I ask.

**Comment by albeanth** [comment]
@dougwoods3  I think that's personal discretion. 

I don't think I will upload my personal info.

**Comment by albeanth** [comment]
@robertschickler your page should be updated. When I click on it the changes you made are there.

**Comment by robertschickler** [comment]
@dougwoods3 Good call, I have reloaded my CV with address blacked out.  My cell phone is relatively public info.  One could find out my address if their Google-fu is sharp.

**Comment by harterj** [comment]
👍  yeah I think I'm gonna do the same for my resume..gonna go black hat

**Comment by robertschickler** [comment]
Thanks you three!  Sorry it took me so long to update it.

**Comment by harterj** [comment]
BTW when I click on your cv link, the header on the file is Elizabeth C Koch.

**Comment by dougwoods3** [comment]
@robertschickler you are far from the last one.

**Comment by robertschickler** [comment]
@harterj Hmmm.  I think she may have authored the original document.  Good catch, lemme see if I can change that.

**Pull request merged**